packages/webpack/src/default-es5-settings.js

```diff
@@ -0,0 +1,41 @@
+// const path = require('path');
+// const HtmlWebpackPlugin = require('html-webpack-plugin');
+// const merge = require('webpack-merge');
+// const defaultSettings = require('./default-settings');
+
+// const defaultOptions = {
+//   index: './index.html',
+// };
+
+// eslint-disable-next-line
+module.exports = userOptions => {
+  return {
+    entry: [
+      require.resolve('@webcomponents/webcomponentsjs/webcomponents-bundle.js'),
+      require.resolve('@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'),
+    ],
+    module: {
+      rules: [
+        {
+          test: /\.m?js$/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: ['@babel/preset-env'],
+              exclude: /(node_modules|bower_components)/,
```

Comment:
`node_modules/lit-html` and `node_modules/lit-element`
need to be transpired right?
for storybook we have something that goes over all excludes and replaces it with an exclude function except this and this...
see: https://github.com/open-wc/open-wc/blob/master/packages/generator-open-wc/generators/publish-storybook/templates/static/.storybook/webpack.config.js#L14